anime-list-master.xml

```diff
@@ -5856,7 +5856,7 @@
   <anime anidbid="1667" tvdbid="79101" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="tt0807703">
     <name>Gekijouban Air</name>
     <mapping-list>
-      <mapping anidbseason="1" tvdbseason="0">;1-3;2-3;3-3;4-3;5-3;6-3;</mapping>
+      <mapping anidbseason="1" tvdbseason="0">;1-1;2-1;3-1;4-1;5-1;6-1;</mapping>
```

Comment:
Only episode 1 of AniDB should be mapped to TVDB special 1, otherwise the AniDB titles for the rest ("Part 1 of 2") would be replaced by TVDB's episode title "Air: The Motion Picture".

That's why I would map the rest (2-6) to episode 0.

Reply:
Updated to map to episode 0